src/multi_dict.jl

```diff
@@ -4,6 +4,7 @@ import Base: haskey, get, get!, getkey, delete!, pop!, empty!,
              insert!, getindex, length, isempty, iterate,
              keys, values, copy, similar,  push!,
              count, size, eltype, empty
+import Base.Iterators: flatten, repeated
```

Comment:
Can we change this to not use `import` and instead overload by qualified name?

Reply:
Oops. thanks. Fixed in latest commit.